docs/jumpstart/week-9.md

```diff
@@ -0,0 +1,403 @@
+ quickly get info about data entries, like average() and max(). As you might expect, Postgres
+  includes this functionality out of the box.
+
+#### Count
+For example, let's say we want to know how many entries are in a table. This is trivial:
+
+```sql
+SELECT COUNT(*) FROM table
+```
+
+The asterisk tells Postgres to count all rows, including `NULL`s and duplicates. We can also add
+ `WHERE` clauses to count only entries that satisfy some condition, like this:
+
+```sql
+SELECT COUNT(*) FROM table WHERE column3 = false
+```
+
+This query returns a table with only a single column `count` and one entry containing the count.
+
+#### Min/Max
+To get the minimum or maximum values of a numeric column in a table, we can use the `MIN()` and `MAX
+()` functions, like this:
+
+```sql
+SELECT MIN(price) FROM products;
+```
+
+Like `COUNT`, this returns a table with a single column `min` and one entry containing the
+ minimum value of the queried column.
+ 
+#### Avg
+Average returns the average of a numeric column:
+
+```sql
+SELECT AVG(price) FROM products
+```
+
+#### Sum
+Sum returns the sum of a numeric column:
+
+```sql
+SELECT SUM(price) FROM products
+```
+
+### Group By & Having
+Let's say we had a `payments` table defined like this:
+
+```sql
+CREATE TABLE payments (
+    id serial PRIMARY KEY,
+    customer_id integer,
+    order_id integer,
+    amount integer
+)
+```
+
+What if we want to know how much each customer has paid over all their orders? We can use SQL's
+ `GROUP BY` clause to group payments by their `customer_id`, and then use `SUM()` to compute the
+  sum.
+  
+```SELECT customer_id, SUM(amount) FROM payments GROUP BY customer_id```
+
+Now, imagine we only cared about our most loyal customers -- the ones who've spent more than $200
+. You might think that we could use a `WHERE` clause to only get the groups of payments that sum
+ to more than 200. But `WHERE` only works for rows, not groups. Instead, we use the `HAVING
+ ` keyword, like this:
+ 
+```SELECT customer_id, SUM(amount) FROM payments GROUP BY customer_id HAVING SUM(amount) > 200```
+
+### Order By
+Often, we'll want to sort the data that we receive from our database. We could do this in our
+ application logic, but it's almost always faster to ask the database to do it directly for us
+ . Fortunately, SQL makes this trivially easy for us.
+ 
+For example, let's say we wanted to get a list of products sorted from cheapest to most expensive
+. This simple query does that for us:
+
+```SELECT * FROM products ORDER BY price ASC;```
+
+Of course `ASC` indicates that the resulting rows will have an ascending price. `DESC` would be
+ the opposite.
+ 
+Sorting works on more than just numeric values. Ordering textual values in `ASC` order sorts by
+ alphabetical order, for example.
+ 
+You can also sort on multiple columns as a tie-breaker by separating each term with a comma:
+
+```SELECT * FROM products ORDER BY column1 ASC, column2 DESC, column3 DESC;```
```

Comment:
MkDocs is a little weird with formatting stuff sometimes. Can you put the backticks on their own line, so

>(triple backtick)
>stuff
>(triple backtick)